src/Callback.php

```diff
@@ -155,12 +134,25 @@ protected function canTerminate(): bool
         return !$reload || $this->owner->name === $reload;
     }
 
+    public function isTriggered(): bool
+    {
+        if ($this->isPostTriggered) {
```

Comment:
I think POST trigger is wrong as we allow nesting for GET only. Can we drop it?

Reply:
Post triggered should be kept. Form submit rely on it. In theory, we should be able to submit any form, using javascript for example, without worrying about the proper GET param that it's needed. 
What is wrong was including the get parameter when Post triggered is required. I have already changed that.
 It should be the user's responsibility to properly set the Post argument when using post triggering.

Reply:
parent callback must always be kept active if child element/View created another one

- with pure POST, this can not be satisfied (as child request may be GET)
- we can keep this but we need to set sticky GET value because of possible nesting (see another comment how)
- then, what is the advantage? Any POST request is GET request + body --> if we use GET only, then we do need this special/extra logic, just generate right callback url for POST form

Reply:
I am not against the idea of keeping GET only. Let's ask @romaninsh what he thinks, as he is the one that created post triggered callback. Maybe it was for a specific reason.